defaults/main.yml

```diff
@@ -1,9 +1,19 @@
 ---
-# The AWS S3 bucket containing the Nessus Debian package file
-package_bucket: "cisa-cool-third-party-production"
+# The processor architecture for which Nessus is to be installed
+architecture: amd64
 
-# The name of the Nessus Debian package file
-package_file: "Nessus-{{ version }}-debian6_amd64.deb"
+# The Linux distribution on which Nessus is to be installed
+distribution: debian6
 
-# The version number of the Nessus Debian package file
-version: "8.15.5"
+# The format/file extension of the desired Nessus package file
+# (e.g. deb, rpm)
```

Comment:
I am reluctant to mention any files other than `deb` until there's testing and support in place for other platforms. There is a `RedHat.yml` vars file but it just points to `Debian.yml` and no testing is done for non-Debian-based platforms.

Reply:
This doesn't resolve this comment, but please see commit 07dff2e where I removed the  `RedHat.yml` file of mysterious provenance.

Reply:
- Please see commit 8065cd2, which removes the need for the user to set the `architecture` and `format` role variables, and thereby removes all mention of them from the README.
- Note also commit 17782be, which generalizes the way that the role was checking whether Nessus was already installed so that it will work for non-Debian distros as well.  I haven't actually added support for non-Debian platforms, but this commit makes that more of a possibility.
- The complication with adding support for non-Debian distributions is that Nessus itself uses different architecture names for different distributions (e.g., `amd64` vs. `x86_64`) _and_ different distribution names depending on the Nessus version (e.g., `debian6` vs. `debian9`).  What is needed is a dictionary that allows one to look up the desired architecture name depending on the `ansible_distribution` fact, and a more complicated dictionary structure (or a function, if I can figure out how to create such a thing in Terraform) to look up the desired distribution name depending on the desired Nessus version, the value of the `ansible_distribution`, and the desired architecture.  It seems simpler to avoid adding support for Fedora, etc. until it is actually needed, especially considering the upkeep required to these data structures every time a new version of Nessus is released.  Are you OK with moving on if I document this in an issue?

Reply:
Yes I was fine with removing any mention of support for non-Debian platforms until we explicitly supported it and just tracking that in an issue.

Reply:
Despite my reluctance to do so, I slowly but surely implemented support for our standard RedHat-based distros.  Can you do a re-review of this PR?